example/tweet-performance-tracker/main.bal

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerina/time;
+import ballerinax/twitter;
+
+configurable string accessToken = ?;
+
+public function main() returns error? {
+
+    // Initialize the client
+    twitter:Client twitter = check new ({
+        auth: {
+            token: accessToken
+        }
+    });
+
+    // Define the username to track and the time range
+    string username = "";
```

Comment:
Why is this variable is assigned to empty string ?

Reply:
My bad, I will assign a value to the variable

Reply:
Solved in : [Fix minor code formatting in examples](https://github.com/ballerina-platform/module-ballerinax-twitter/pull/111/commits/c955f46ceefb7d53336e8f627f62a7edc126e76d)